lora/message.go

```diff
@@ -0,0 +1,43 @@
+package lora
+
+// RxInfo receiver parameters
+type RxInfo []struct {
```

Comment:
Why do we copy these? Can't we just import packages where these are defined from LoRa Server?

Reply:
As we discuss, they are in `internal` lora app server repo. We keep this structures.